Heroes.SDK.Library/Classes/NativeClasses/TObjTeam.cs

```diff
@@ -1,10 +1,161 @@
 ﻿using Reloaded.Hooks.Definitions;
 using Reloaded.Hooks.Definitions.X86;
+using System.Runtime.InteropServices;
+using Heroes.SDK.Definitions.Structures.RenderWare;
+using Heroes.SDK.Definitions.Enums;
+using Heroes.SDK.Definitions.Structures.Input;
 
 namespace Heroes.SDK.Classes.NativeClasses
 {
+    [StructLayout(LayoutKind.Explicit, Pack = 4, Size = 0x27C)]
     public unsafe struct TObjTeam
     {
+        [FieldOffset(0x0)]
+        public TObject tobj;
+        [FieldOffset(0x28)]
+        public fixed byte gap0[4];
+        [FieldOffset(0x2C)]
+        public int pointer_0x2c;
+        [FieldOffset(0x30)]
+        public byte field_30;
+        [FieldOffset(0x34)]
+        public ENUM_TEAM_KIND teamKind;
+        [FieldOffset(0x38)]
```

Comment:
Consider merging this structure with the existing one.

https://github.com/Heroes-Hacking-Central/Heroes.SDK/blob/master/Heroes.SDK.Library/Definitions/Structures/Player/TeamTop.cs

You can keep the new name from disassembly, but copy the docs and the few fields that are better defined.

Reply:
Done https://github.com/Heroes-Hacking-Central/Heroes.SDK/pull/3/commits/12ad38812701598badd251ae0f883d5123b0c84e